### DISCUSSION TITLE
August community call

### DISCUSSION THREAD

**Discussion by iason9** [EddieHubCommunity/support#2260] [Community Call]
Welcome to the EddieHub Community Call for August!

Asking questions/discussing things -

Start a new thread, and we can get responses on both the call and inside this github Discussion :)
You can drop an intro in the introduction thread below :)

**Actions**
- Live with @SaraJaoude on reviewing resumes and CVs
- Discord quiz game

**Comment by eddiejaoude** [comment]
Please introduce yourself and include links 👇

**Reply by rushikesh-rd** [reply]
hello, I'm rushikesh currently pursuing masters in computer application. And looking forward to contributing to open source.[Twiter](https://twitter.com/rushikeshDhus) [Linkedin](https://www.linkedin.com/in/rushikesh-dhus-426713171/)

**Reply by ashwinexe** [reply]
Hello, I am Ashwin! 👋🏻 I am a engineering student from India. Super interested in Open Source, Hackathons and Communities. 
Would love to connect: https://bio.link/ashwinexe

**Reply by guptasajal411** [reply]
Hey, I am Sajal. I am a sophomore CS student from India. I am currently learning Backend Development, specifically REST API's. Catch me on [LinkedIn](https://www.linkedin.com/in/sajalgupta20/), [Twitter](https://twitter.com/guptasajal411) and, well, GitHub!

**Reply by divyanshraj0408** [reply]
Hey, I am Divyansh Raj! I am an Engineering Student from India. I am excited about Open Source, hackathons and communities. 
catch me on [Lineked.in](https://www.linkedin.com/in/divyansh-raj/).
You people are doing great work.

**Comment by eddiejaoude** [comment]
Q. If you had a "magic wand" what would you wish for EddieHub so it can help you?

**Reply by ashwinexe** [reply]
Add million more swags to the EddieHub catalog.

**Reply by naomi-lgbt** [reply]
- Money to pay for my servers 😛 
- A scholarship for folks who are active in our community to provide financial support for continued open source development

**Reply by iason9** [reply]
a recognized certification that's very valuable on the tech industry and in the eyes of big tech companies.

**Reply by kkhitesh** [reply]
Some sort of guide to freelancing like writing effective cover letters, getting more clients

**Comment by ashwinexe** [comment]
Q. What is the routine taught to you in school/college you think is making you counterproductive?

**Reply by iason9** [reply]
waiting to learn everything rather just start using what you need right away

**Comment by naomi-lgbt** [comment]
What other games would you like to see for game night?

**Reply by eddiejaoude** [reply]
Community build game

**Reply by naomi-lgbt** [reply]
My own thoughts are the Jackbox games, Gartic Phone, maybe Minecraft (but it's not free 🙁 )

**Reply by eddiejaoude** [reply]
Battlesnake

**Reply by iason9** [reply]
Minecraft is awesome if we get to do it

**Reply by ashwinexe** [reply]
> My own thoughts are the Jackbox games, Gartic Phone, maybe Minecraft (but it's not free 🙁 )

I love Jackbox games! QuipLash is very good

**Reply by Vyvy-vi** [reply]
Minecraft!!!

**Reply by schmelto** [reply]
Cards against downtime (don't know if this is possible online)

**Reply by guptasajal411** [reply]
Valorant! It's really fun. Only 10 people can play at a time, while others can spectate and do commentary on the game.

**Reply by ashwinexe** [reply]
Skribble/Sketchful.io
Geoguesser

**Reply by kkhitesh** [reply]
Brawlhalla kinda like super smash bros

**Reply by guptasajal411** [reply]
**[GeoGuessr](https://www.geoguessr.com/)** is pretty cool!

**Comment by eddiejaoude** [comment]
Q. What discord bot games would you like to see?

**Reply by schmelto** [reply]
Can we include battlesnake 😂

**Reply by eddiejaoude** [reply]
@schmelto I added that now to @nhcarrigan 's thread 😂

**Reply by kalash99** [reply]
> Q. What discord bot games would you like to see?

A simple Tic Tak Toe game can be a good start!

**Reply by shreemaan-abhishek** [reply]
Stone Paper Scissors? :stuck_out_tongue_closed_eyes:

**Reply by Vyvy-vi** [reply]
Eddie-Snake

**Reply by prateek796** [reply]
Can you add roulette to it ? It would be fun imagining us playing casino in Las Vegas🤪

**Reply by eddiejaoude** [reply]
@Vyvy-vi suggestion quiz game

- json for types of questions and levels

```ts
// javascript questions/answers 
[
   {
       level: 1,
       question: "question 1",
       answer: "answer 1"
]
```

**Reply by kkhitesh** [reply]
I saw a Pokemon game on some server random monsters woul d spawn and user catch em and stuff

**Comment by ashwinexe** [comment]
Q. What is the one skill you'd like to unlearn? (i.e. relearn from scratch because you might want to learn it the right way)

**Reply by Vyvy-vi** [reply]
I don't think of this as unlearn, more like re-learn... I want to re-learn OOP, and DSA with a language like C or Java..
I think I didn't learn it properly with python

**Reply by iason9** [reply]
there is no wrong and right way to learn, just learn as you go and eventually you will adabt to the good practices and build on that, there is actually bad and good habits that you can unlearn and learn

**Reply by naomi-lgbt** [reply]
The only wrong way to learn is to stop learning :)

**Reply by ashwinexe** [reply]
I would like to re-learn firebase. There's common emotion attached with re-learning from scratch hence we don't tend to try it even if we can (mostly because we've to start from scratch again)

**Comment by Vyvy-vi** [comment]
Q. Freelancing Question - Say you worked for someone, but a dependency broke... You offered to fix it, but then they offer you to pay again, even though you refused.. What do you do in this situation, and how do you refuse more payments?

**Reply by naomi-lgbt** [reply]
Take the money 🤷🏻 

If you were contracted to build a project, you built that project. A dependency breaking and requiring an update/refactor would be maintenance not covered in the original scope of work (unless the contract states that routine maintenance is expected, in which case you should have negotiated a monthly retainer).

**Reply by guptasajal411** [reply]
How do you get started with Freelancing? I have no experience at all

**Reply by Zain-Muiz** [reply]
How thoughtfullll @Vyvy-vi 🥺, But where do wee find such clients ??😜😂  I remember I had the same issue few months ago, and my client refused to pay the remaining balance. I just got the pay from advance payment. :(

**Reply by Vyvy-vi** [reply]
Well, someone in our community introduced me to them :)

**Reply by eddiejaoude** [reply]
If there are changes to dependencies, you should charge for extra work

**Comment by kaiwalyakoparkar** [comment]
Q. I was thinking of getting into freelancing, so what are the platforms you would recommend?

**Reply by Vyvy-vi** [reply]
I think it depends 🤔 
I've had one request from Upwork, and multiple opportunities for freelancing(sorta) from community work that I did, which led to people suggesting other people to get help from me

**Reply by shreemaan-abhishek** [reply]
upwork.com, fiverr.com, freelancer.com. These are the websites I have heard nice things about.

**Reply by eddiejaoude** [reply]
Local small companies is a great place to start

**Reply by eddiejaoude** [reply]
I would suggest going on all planforms and seeing which one takes off and focus more on that platform - from my limited experience using these platforms, Upwork seemed more professional

**Comment by Vyvy-vi** [comment]
Q) How do you move towards writing better tests?

**Reply by naomi-lgbt** [reply]
```js
assert.isTrue(true);
```

**Reply by eddiejaoude** [reply]
what type of tests? Frontend, backend or both?

**Reply by eddiejaoude** [reply]
- Backend: We have some examples in our API project http://github.com/EddieHubCommunity/api
- Frontend: I would use Cypress
- Discord bot: uses web sockets so difficult to test

**Comment by ashwinexe** [comment]
Do you get time to work on your extra-skills/Interests after getting job? How is it compared to time you used to get when you were a student?

**Reply by eddiejaoude** [reply]
It depends on you and the company, I would recommend to always be learning but make sure you get some rest for your mental health

It is a marathon not a sprint

**Comment by divyanshraj0408** [comment]
I am new into programming, i know Java and basics of Data Structures and algorithms(I am in first year of my college), so i wanted to know that, is my current experience enough to get me started with open source or should i learn more and then come back?

**Reply by Vyvy-vi** [reply]
Yes, it's sufficient!

**Reply by eddiejaoude** [reply]
yes yes yes 👍 please contribute to open source 🤓

**Reply by kaiwalyakoparkar** [reply]
You can get started with the open source even if you don't know any programming language

**Reply by iason9** [reply]
Yes of course!

**Reply by eddiejaoude** [reply]
You can add so much value, learn and contribute in public 🎉

**Reply by Zain-Muiz** [reply]
Guess what ? @divyanshraj0408 You have already started  ! Now its all about sticking here, and keep going ! 🚀 Everyone at EddieHub has got your back, Just reach out ! ✌ Dont let anything make you give up on collaborating and contributing !  You're on the right path 😁 Looking forward to see you more around.

**Comment by Mohit15985001** [comment]
Hello My name is Mohit... linked in : https://www.linkedin.com/in/mohit-nathwani-425214191/
how do i reach out to new people when i am getting started with open source & basically under about the project team is working on?

**Reply by Zain-Muiz** [reply]
Heyy @Mohit15985001 , When you go to the repository of the project you want to contribute, You will most probably see a Slack channel (If you dont know what slack is, its similar to discord) or any other social channels. Join in those. Most of the communities have new comer places in their social places. And engage, just like you did today !

**Reply by kaiwalyakoparkar** [reply]
You can do following things
- Opening a discussion for interacting with team
- If discussion is not enabled then you can open an issue for conversation.
- You should definitely their mailing list (discord, slack, zulip) where people can help you personally and more interactively

**Reply by eddiejaoude** [reply]
Get started in the discussion and see where it takes you. You can add value, by adding value to the project, the maintainers will want to help you get involved more and more

**Comment by kaiwalyakoparkar** [comment]
Should I learn multiple programming languages once or should I go with a single language and move ahead with it.

**Reply by Chandan-h-509** [reply]
Hey @kaiwalyakoparkar . That's a debatable question.
That totally depends on the tech stack you want to work. I use Java for DSA and Python for ML and DL.
But if you are new to programming it would be better for you to learn a single language initially and then you can learn other languages based on your tech stack.
